src/ui/public/agg_types/metrics/top.js

```diff
@@ -0,0 +1,59 @@
+import { noop, get } from 'lodash';
+import MetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import topEditor from 'ui/agg_types/controls/top.html';
+
+export default function AggTypeMetricTopProvider(Private) {
+  const MetricAggType = Private(MetricAggTypeProvider);
+
+  return new MetricAggType({
+    name: 'top_hits',
+    title: 'Top',
+    makeLabel: function (aggConfig) {
+      const prefix = aggConfig.params.sortOrder.val === 'desc' ? 'Latest' : 'Earliest';
```

Comment:
Well there's already a custom label field, I think we just need a better default. I was almost sold on Latest/Earliest but then I realized it would be especially silly for non-timebased index patterns. How about something really generic, like Top/Bottom?